cmd/generate/codeowners/output.go

```diff
@@ -3,23 +3,37 @@ package codeowners
 import (
 	"fmt"
 	"os"
+	"path/filepath"
 	"regexp"
 	"sort"
 	"strings"
 
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
 	"github.com/open-sauced/pizza-cli/pkg/config"
 )
 
-func generateOutputFile(fileStats FileStats, outputPath string, ownersStyleFile bool, config *config.Spec) error {
+func generateOutputFile(fileStats FileStats, outputPath string, opts *Options, cmd *cobra.Command) error {
 	// Open the file for writing
 	file, err := os.Create(outputPath)
 	if err != nil {
 		return fmt.Errorf("error creating %s file: %w", outputPath, err)
 	}
 	defer file.Close()
+	var flags []string
+
+	cmd.Flags().Visit(func(f *pflag.Flag) {
+		flags = append(flags, fmt.Sprintf("--%s %v", f.Name, f.Value.String()))
+	})
+	generatedCommand := fmt.Sprintf("# $ pizza generate codeowners %s/ ", filepath.Base(opts.path))
```

Comment:
Nit: remove extra space at the end of this generated command which can show up as trailing whitespace if no flags are given:

```suggestion
	generatedCommand := fmt.Sprintf("# $ pizza generate codeowners %s/", filepath.Base(opts.path))
```

Then to get this to work when flags are present, I'd just add the trailing space there:

```go
	if len(flags) > 0 {
		generatedCommand += " "
		generatedCommand += strings.Join(flags, " ")
	}
```